test/test_rules_patterns.py

```diff
@@ -1,7 +1,133 @@
 # -*- coding: utf-8 -*-
+import os
+
+import pytest
+
 from .helper import check_evaluation
 
 
+DEBUGRULESPAT = int(os.environ.get("DEBUGRULESPAT", "0")) == 1
```

Comment:
Still there are a part of the behaviour that I did not cover. This allows to choose mark the corresponding tests as xfail or skip, as in other unit tests.